src/framework/ui/ikeynavigationsection.h

```diff
@@ -0,0 +1,53 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2021 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+#ifndef MU_UI_IKEYNAVIGATIONSECTION_H
+#define MU_UI_IKEYNAVIGATIONSECTION_H
+
+#include <QString>
+#include <QList>
+
+namespace mu::ui {
+class IKeyNavigationSubSection
```

Comment:
Do we really need this interface? It seems that we only need IKeyNavigationSection. Technically a situation may happen when we need sub-sub sections and I afraid that we'll create IKeyNavigationSubSubSection :)

Reply:
I thought :) And I decided to add a separate interface, for two purposes:
1. Avoid `section-> section-> section...` tree. We only have a `section-> subsection-> control` and that's it.
2. For cleaning type

Reply:
Make sense, but I don't like that we have two almost identical interfaces. Maybe is it better to create one base interface and inherit IKeyNavigationSection from it?

Reply:
I think this is overengineering :) [KISS](https://en.wikipedia.org/wiki/KISS_principle)

Reply:
In the next PR I going to add common interface